code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1090,14 +1090,14 @@ All effects don't start immediately, but rather get worse over time; the rate is
 
 /datum/reagent/consumable/ethanol/irishcarbomb
 	name = "Irish Car Bomb"
-	description = "Mmm, tastes like chocolate cake..."
+	description = "Mmm, tastes like Anger."
 	color = "#2E6671" // rgb: 46, 102, 113
 	boozepwr = 25
 	quality = DRINK_GOOD
-	taste_description = "delicious anger"
+	taste_description = "the spirit of Ireland"
```

Comment:
you replaced the description to tastes like anger but you removed it tasting like anger

Reply:
> 
> 
> you replaced the description to tastes like anger but you removed it tasting like anger

my bad, I fixed it